File: CodeSamples/csharp/src/IO.Swagger/Model/TerminalSettings.cs
using System;
using System.Linq;
using System.IO;
using System.Text;
using System.Text.RegularExpressions;
using System.Collections;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Runtime.Serialization;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;
using System.ComponentModel.DataAnnotations;
using SwaggerDateConverter = IO.Swagger.Client.SwaggerDateConverter;

namespace IO.Swagger.Model
{
    [DataContract]
    public partial class TerminalSettings :  IEquatable<TerminalSettings>, IValidatableObject
    {
        public TerminalSettings(int? iD = default(int?), int? terminalID = default(int?), string sECCode = default(string), bool? gatewayTerminal = default(bool?), bool? consumerCredits = default(bool?), bool? dLRequired = default(bool?), bool? checkVerify = default(bool?), bool? identityVerify = default(bool?))
        {
            this.ID = iD;
            this.TerminalID = terminalID;
            this.SECCode = sECCode;
            this.GatewayTerminal = gatewayTerminal;
            this.ConsumerCredits = consumerCredits;
            this.DLRequired = dLRequired;
            this.CheckVerify = checkVerify;
            this.IdentityVerify = identityVerify;
        }

        [DataMember(Name="ID", EmitDefaultValue=false)]
        public int? ID { get; set; }

        [DataMember(Name="Terminal_ID", EmitDefaultValue=false)]
        public int? TerminalID { get; set; }

        [DataMember(Name="SEC_Code", EmitDefaultValue=false)]
        public string SECCode { get; set; }

        [DataMember(Name="Gateway_Terminal", EmitDefaultValue=false)]
        public bool? GatewayTerminal { get; set; }

        [DataMember(Name="Consumer_Credits", EmitDefaultValue=false)]
        public bool? ConsumerCredits { get; set; }

        [DataMember(Name="DL_Required", EmitDefaultValue=false)]
        public bool? DLRequired { get; set; }

        [DataMember(Name="Check_Verify", EmitDefaultValue=false)]
        public bool? CheckVerify { get; set; }

        [DataMember(Name="Identity_Verify", EmitDefaultValue=false)]
        public bool? IdentityVerify { get; set; }
    }
}

File: CodeSamples/nodejs/src/model/TerminalSettings.js
/* eslint-disable no-prototype-builtins */
import ApiClient from '../ApiClient';

export default class TerminalSettings {
  constructor() {
    this.ID = 0;
    this.Terminal_ID = 0;
    this.SEC_Code = '';
    this.Gateway_Terminal = false;
    this.Consumer_Credits = false;
    this.DL_Required = false;
    this.Check_Verify = false;
    this.Identity_Verify = false;
  }

  static constructFromObject(data, obj) {
    if (data) {
      obj = obj || new TerminalSettings();

      if (data.hasOwnProperty('ID')) {
        obj['ID'] = ApiClient.convertToType(data['ID'], 'Number');
      }
      if (data.hasOwnProperty('Terminal_ID')) {
        obj['Terminal_ID'] = ApiClient.convertToType(data['Terminal_ID'], 'Number');
      }
      if (data.hasOwnProperty('SEC_Code')) {
        obj['SEC_Code'] = ApiClient.convertToType(data['SEC_Code'], 'String');
      }
      if (data.hasOwnProperty('Gateway_Terminal')) {
        obj['Gateway_Terminal'] = ApiClient.convertToType(data['Gateway_Terminal'], 'Boolean');
      }
      if (data.hasOwnProperty('Consumer_Credits')) {
        obj['Consumer_Credits'] = ApiClient.convertToType(data['Consumer_Credits'], 'Boolean');
      }
      if (data.hasOwnProperty('DL_Required')) {
        obj['DL_Required'] = ApiClient.convertToType(data['DL_Required'], 'Boolean');
      }
      if (data.hasOwnProperty('Check_Verify')) {
        obj['Check_Verify'] = ApiClient.convertToType(data['Check_Verify'], 'Boolean');
      }
      if (data.hasOwnProperty('Identity_Verify')) {
        obj['Identity_Verify'] = ApiClient.convertToType(data['Identity_Verify'], 'Boolean');
      }
    }
    return obj;
  }
}
